src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background-color: black;
+  margin: 0;
+  padding: 20px;
+}
+
+.blue-box {
+  width: 350px;
+  background-color: #308dc0;
+  margin-bottom: 40px;
+}
+
+.yellow-box {
+  background-color: #ffa500;
+}
+
+.red-border {
+  border-color: #e75551;
+  border-style: solid;
+  border-width: 1px;
+  padding: 10px;
+  box-sizing: border-box;
+}
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.